docs/test_the_installation.rst

```diff
@@ -71,13 +71,13 @@ Test the web interfaces
    - Usage of the Source Interface is covered by our :doc:`Source User
      Manual <source>`.
 
-#. Test that you can access the Journalist Interface, and that you can log
+#. Test that you can access the *Journalist Interface*, and that you can log
    in as the admin user you just created.
 
    - Open the Tor Browser and navigate to the onion URL from
-     app-journalist-aths. Enter your passphrase and two-factor
+     ``app-journalist-aths``. Enter your passphrase and two-factor
      authentication code to log in.
-   - If you have problems logging in to the Admin/Journalist Interface,
+   - If you have problems logging in to the *Admin/Journalist Interface*,
```

Comment:
I think this is the only occurrence of "Admin Interface", so I think it should be removed to just say "Journalist Interface."

Reply:
Updating this comment to say that the docs use the term `Admin Interface` often enough that this wouldn't appear out of place.